packages/grafana-ui/src/components/uPlot/utils.ts

```diff
@@ -106,6 +106,57 @@ export function collectStackingGroups(f: Field, groups: Map<string, number[]>, s
   }
 }
 
+/**
+ * Finds y axis midpoind for point at given idx (css pixels relative to uPlot canvas)
+ * @internal
+ **/
+
+export function findMidPointYPosition(u: uPlot, idx: number) {
```

Comment:
Same with testing this, what to do with uPlot as a dependency in the unit test? 😬